README.md

```diff
@@ -82,6 +82,7 @@ directory in order for Grabbit to be built from source.
 6. Enter the path to the Grabbit config you created in step. Screenshot of how it will look like : ![alt text](assets/GrabbitConfig.jpg)
 7. Once the Grabbit content sync is kicked off, you will get a confirmation of the kicked off jobs like : ![alt text](assets/GrabbitComplete.jpg)
 
+**Note:** The client and server side credentials entered above will be used as the credentials when creating the JCR session on both Client and Server.
 
```

Comment:
Normally I'm all for explicit. In this case I think the context makes it clear enough and this just muddies the water.